Rubberduck.Resources/Inspections/InspectionInfo.resx

```diff
@@ -392,6 +392,15 @@ If the parameter can be null, ignore this inspection result; passing a null valu
     <value>A class module that is meant to be used as interface for concrete classes should generally be abstracted of any implementations. If it is your intention to use this class module as a concrete, you can safely ignore this inspection result.</value>
   </data>
   <data name="ArgumentWithIncompatibleObjectTypeInspection" xml:space="preserve">
-    <value>The VBA compiler does not raise an error if an object is passed as an argument for a parameter with an incompatible declared object type, i.e. with an object type that is neither the same type, a supertype nor a subtype. Under almost all circumstances passing such an argument leads to a run-time error, which is harder to detect and indicates a bug. In all other situations the code can be changed to use only assignments between compatible declared types.</value>
+    <value>The VBA compiler does not raise an error if an object is passed as an argument for a parameter with an incompatible declared object type, i.e. with an object type that is neither the same type, a supertype nor a subtype. Under almost all circumstances passing such an argument leads to a run-time error, which is harder to detect and indicates a bug. In all other situations the code can be changed to only pass arguments of compatible declared types.</value>
+  </data>
+  <data name="ValueRequiredInspection" xml:space="preserve">
+    <value>The VBA compiler does not raise an error if an object is used in a place that requires a value type and the object's declared type does not have a suitable default member. Under almost all circumstances, this leads to a run-time error, which is harder to detect and indicates a bug.</value>
+  </data>
+  <data name="ProcedureRequiredInspection" xml:space="preserve">
+    <value>The VBA compiler does not raise an error if an object variable is used in a place that requires a procedure and the object's declared type does not have a suitable default member. Under almost all circumstances, this leads to a run-time error, which is harder to detect and indicates a bug.</value>
+  </data>
+  <data name="DefaultMemberRequiredInspection" xml:space="preserve">
+    <value>The VBA compiler does not raise an error if an indexed default member call is required but the object's declared type does not have a suitable default member. Under almost all circumstances, this leads to a run-time error, which is harder to detect and indicates a bug.</value>
```

Comment:
The run-time error would be 438 "Object does not support property or method" every time, no? I can't help thinking any relevant error code(s)/number(s) should be somewhere in the inspection info.. like when a user *does* get that error, it would be nice if they/we could just type "438" in the inspection toolwindow's search bar, and one of these inspection results will come up and explain what's going on - just like what I'm reading here; I would only reference the relevant error number(s) somewhere ;-)

Reply:
Added the info about possible error numbers.

In all cases it is either error 91 or 438 depending on whether the object is `Nothing` or not.